app/Main.hs

```diff
@@ -265,6 +270,28 @@ generateSite siteDirRaw flakyPkgs reportsDir = do
   Site.generateSite params
   putStrLn "Done."
 
+-- | Check whether we've already seen this target\/commit combo. (If so, it
+-- makes sense for the CI script to skip building a build plan again).
```

Comment:
Isn't this documentation misguiding? The function only checks if the newest item in the history has the same attributes (whatever they are) as the provided metadata info.

The documentation of the function and the command line argument need to be more precise.